.github/workflows/build_and_test_with_resty_events.yml

```diff
@@ -1,6 +1,13 @@
-name: Build and test
-
-on: [push, pull_request]
+name: Build and test - with resty_events
+
+on:
```

Comment:
The previous trigger made running the workflow twice if there was a PR opened - so I've copied the options that we have in `lint.yml`

Reply:
fyi; you might want to add concurrency in there, example: https://github.com/Tieske/project.lua/blob/main/.github/workflows/unix_build.yml#L3-L16

(just a consideration, no blocker)